src/controllers/controller.bar.js

```diff
@@ -87,7 +87,10 @@ defaults._set('horizontalBar', {
 
 			label: function(item, data) {
 				var datasetLabel = data.datasets[item.datasetIndex].label || '';
-				return datasetLabel + ': ' + item.xLabel;
+
```

Comment:
you didn't need to change this section of code. it was more concise before

Reply:
done